.github/workflows/pr-realm-js.yml

```diff
@@ -329,6 +329,14 @@ jobs:
         if: ${{ matrix.variant.environment == 'react-native' }}
         run: ccache --set-config="compiler_check=content"
 
+      # Hermes doesn't work with Cocoapods 1.15.0
+      # https://forums.developer.apple.com/forums/thread/745518
+      - name: Install older Cocoapods version
+        if: ${{ matrix.variant.os == 'ios') }}
```

Comment:
I reverted this with https://github.com/realm/realm-js/commit/ec843dadb4ca2b15241caf202b74350e11ac64e8